pages/posts/flight-of-a-byte.tsx

```diff
@@ -0,0 +1,112 @@
+
+                We&apos;ve mentioned the traditional networking code “under the hood” to remove some of the mystery. But that code doesn&apos;t immediately read the message from <code>client2</code>. Instead, there are some communication steps which allow Wesh to use libp2p to operate in a peer-to-peer environment.
+                </p>
+                
+                <h3>Message log and “head”</h3>
+                <p>
+                To send a message in traditional networking, one user connects to the network address of the other user and transmits the message. Or the user connects to a central server and appends to the message log there, while the other user connects to the same central server and reads the message log. But this doesn&apos;t work in a peer-to-peer environment. Instead, the goal is for each group member to have a copy of the message log, where there is a “head” (the latest message) which refers to all the previous messages in a chain. To send a message to the group, a member adds to the local copy of the message log and updates the “head”. This will be synchronized with the other members.
+                </p> 
+                <p>
+                 As mentioned, <code>client2</code> doesn&apos;t send the message “Hello” directly to the other group members. Instead, it puts the message in its own message log and updates the head. The other members learn about the new head which means there is a new message to fetch.
+                </p>
+                <h3>libp2p pubsub</h3>
+                <p>
+                 So now the question is, how does <code>client1</code> learn that <code>client2</code> says there is a new message log head? This uses libp2p pubsub where peers can subscribe to a topic and publish a pubsub message to that topic, or receive pubsub messages. (We say “pubsub message” to not confuse with the ordinary messages in the log which we&apos;ll discuss below.)
+                </p>
+                <p>
+                libp2p pubsub is a powerful tool with algorithms which allow it to scale to many peers and to make sure they all get the pubsub message. You can read more in its documentation, but for this blog post we only need to say that the Wesh group has a libp2p topic name like 
+                &nbsp;<code>
+                /orbitdb/bafyreihpdptqwgpuwu4ob6nusxebmdx6fm6mq5pma5hva6eu5u74pflo7e/2ac4e88c5272f900c76fd36989f9780e5b2c95d75d38fe5bcef0345b34bc4806_wesh_group_messages
+                    </code>
+                &nbsp;a pubsub message to this topic that contains the message log head. The libp2p on <code>client1</code> receives this pubsub message and sends it (code) internally to all processes which are subscribed to the topic. The Wesh application of <code>client1</code> has used ActivateGroup and is therefore subscribed to this topic and receives the pubsub message (code).
+                </p>
```

Comment:
"a pubsub message to this topic" should be "Client2 publishes a pubsub message to this topic."

Reply:
Also, please include the code links for "sends it (code)" and "pubsub message (code)". See the Notion draft.

Reply:
Still need to fix this:
"a pubsub message to this topic" should be "Client2 publishes a pubsub message to this topic."